pytest_mitmproxy.py

```diff
@@ -1 +1,106 @@
 # -*- coding: utf-8 -*-
+import re
+import os
+import six
+import time
+import shutil
+import pytest
+from contextlib import contextmanager
+from tempfile import TemporaryFile
+if six.PY3:
+    from urllib.parse import urlparse
+    from subprocess import Popen, CalledProcessError, TimeoutExpired
+elif six.PY2:
+    from urlparse import urlparse
+    from subprocess import Popen, CalledProcessError
```

Comment:
It might make more sense to insist on subprocess32 on python2. Some of the timeout stuff isn't available until python3.